### ISSUE TITLE
Request: The ability to deploy this with portainer.io

### ISSUE THREAD

**Issue by Altonymous** [ryanwi/rails7-on-docker#192]
It would be great if people could directly deploy this repository with portainer.io.

**Comment by Altonymous** [comment]
Created this fork.  Not able to open a PR for some reason.

https://github.com/Altonymous/rails7-on-docker

**Comment by ryanwi** [comment]
Good idea, I'll check out your repo. Do you have a guide you like? I haven't used portainer much yet, but am interested in exploring.

It looks like that when a repo is created from a GitHub template, it doesn't support the GitHub flow for pull requests. That's a bummer.

![Screenshot 2022-11-06 at 9 49 20 AM](https://user-images.githubusercontent.com/49515/200186634-32af4a73-7702-45c1-9b94-96f82858c057.png)

you may have to fork this repo to be able to create a PR for it.

**Comment by ryanwi** [comment]
confirmed, sorry about that.

> Branches created from a template have unrelated histories, so you cannot create pull requests or merge between the branches.

Branches created from a template have unrelated histories, so you cannot create pull requests or merge between the branches.
